inc/common.class.php

```diff
@@ -621,9 +630,9 @@ public static function processMassiveActionsForOneItemtype(
                 }
```

Comment:
Could you add an icon for the single massive action?
see : https://github.com/glpi-project/glpi/pull/20113